boards/arm/frdm_k64f/pinmux.c

```diff
@@ -39,6 +39,14 @@ static int frdm_k64f_pinmux_init(struct device *dev)
 	pinmux_pin_set(portb, 17, PORT_PCR_MUX(kPORT_MuxAlt3));
 #endif
 
+#ifdef CONFIG_UART_MCUX_2
+	/* UART2 RX, TX */
+	pinmux_pin_set(portd, 0, PORT_PCR_MUX(kPORT_MuxAlt3));
+	pinmux_pin_set(portd, 1, PORT_PCR_MUX(kPORT_MuxAlt3));
+	pinmux_pin_set(portd, 2, PORT_PCR_MUX(kPORT_MuxAlt3));
+	pinmux_pin_set(portd, 3, PORT_PCR_MUX(kPORT_MuxAlt3));
```

Comment:
You're configuring pinmuxes for RTS and CTS, but we don't have hardware flow control support in the uart driver yet. Do you need it?

Reply:
The Starter Kit Hardware Guide (v1.7) states that all 4 pins need to be in UART mode (MuxAlt3), however the ARM Mbed examples I looked at only setup the TX,RX pins for UART use.  And my testing hasn't been effected by the fact that HW flow control isn't supported in our driver.

I guess for this change, I'm just being thorough.